src/components/NormalLeftRow/index.jsx

```diff
@@ -50,6 +51,17 @@ function NormalLeftRow({ classes, treeNode, refType, setSchemaTree }) {
     const bracketTypes = [...NESTED_TYPES, 'closeArray', 'closeObject'];
     const combinationTypes = [...COMBINATION_TYPES, 'and', 'or', 'nor'];
 
+    /**
+     * If type is not specified, create a tooltip with an icon.
+     */
+    if (!type) {
+      return (
+        <Tooltip title={TOOLTIP_DESCRIPTIONS.noType}>
+          <WarningIcon color="inherit" />
+        </Tooltip>
+      );
+    }
```

Comment:
Adding an example in the storybook where we can see this show up could be good.

Reply:
Added example 👍